### PR TITLE
added currency converter

### PR DESCRIPTION
**Pull request by denysM789**

### CODE REVIEW

File: src/components/Coinpage/BottomChart/BottomChart.js [hunk 1]
```diff
@@ -1,39 +1,126 @@
-import React from "react";
-import DurationSelector from "../DurationSelector/DurationSelector";
+import React, { useEffect, useState } from "react";
+
```
Comment (yoloOnTheBattlefield) [inline src/components/Coinpage/BottomChart/BottomChart.js:2]: no new line

File: src/components/Coinpage/CoinInfo/CoinInfo.js [hunk 2]
```diff
@@ -83,22 +83,47 @@ const CoinInfo = (props) => {
   const url3 = coin?.links.blockchain_site[1];
   const url4 = coin?.links.blockchain_site[2];
```
Comment (yoloOnTheBattlefield) [inline src/components/Coinpage/CoinInfo/CoinInfo.js:84]: const [,url3, url4] = coin?.links.blockchain_site
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/Coinpage/BottomChart/BottomChart.js (hunk 1 above)
Comment (denysM789) [reply src/components/Coinpage/BottomChart/BottomChart.js:2]: done, thanks bro

File: src/components/Coinpage/CoinInfo/CoinInfo.js (hunk 2 above)
Comment (denysM789) [reply src/components/Coinpage/CoinInfo/CoinInfo.js:84]: nice! done, thanks bro

**Pull request merged**